README.md

```diff
@@ -2,9 +2,9 @@
 
 Presto is a distributed SQL query engine for big data.
 
-See the [Presto installation documentation](https://prestodb.github.io/docs/current/installation) for deployment instructions.
+See the [Presto installation documentation](https://prestodb.io/docs/current/installation) for deployment instructions.
```

Comment:
```suggestion
See the [Presto installation documentation](https://prestodb.io/docs/current/installation.html) for deployment instructions.
```
https://prestodb.io/docs/current/installation fails, returning a 403 Forbidden. Suggest this link.